tfhe/src/shortint/ciphertext/mod.rs

```diff
@@ -9,10 +9,50 @@ use std::fmt::Debug;
 
 use super::parameters::{CiphertextConformanceParams, CiphertextListConformanceParams};
 
-/// This tracks the number of operations that has been done.
+/// This tracks the max degree that can be in a Ciphertext.
```

Comment:
Was this change meant ? I'm worried the using `max degree` might create confusions with the real max degree

Reply:
I think the docstring is not correct, we want the degree in the ct

Reply:
Indeed max degree is a ambiguous name.
IMO the name `degree` is ambiguous itself because it's not well defined (setting the degree to `msg_modulus-1` after a comparison instead of `1` is correct even if not optimal)

What do you suggest replacing this comment by?
Maybe we could add a paragraph explaining what is the degree and the subtleties

Reply:
One point is that having the noise separated from the degree, is that after comparisons degree can be set to 1.

I don't think its necessary to change the docstring of degree now, it would make sense once the noise integration is done. Then it should be simpler to explain as degree would just be the max possible value

Reply:
Reverted